crates/runtime/src/dataconnector/iceberg.rs

```diff
@@ -300,4 +240,80 @@ impl DataConnector for IcebergDataConnector {
 
         Ok(Arc::new(table_provider))
     }
+
+    async fn load_hadoop_catalog(
```

Comment:
There are no changes in `load_hadoop_catalog`, it is incorrectly showed as modified as I've moved `read_provider` into connector as `create_iceberg_table_provider`